src/features/game/lib/constants.ts

```diff
@@ -41,40 +42,91 @@ export const CHICKEN_POSITIONS: ChickenPosition[] = [
   { top: GRID_WIDTH_PX * 1.59, right: GRID_WIDTH_PX * 14.12 },
 ];
 
-export const INITIAL_STOCK: Inventory = {
-  "Sunflower Seed": new Decimal(400),
-  "Potato Seed": new Decimal(200),
-  "Pumpkin Seed": new Decimal(150),
-  "Carrot Seed": new Decimal(100),
-  "Cabbage Seed": new Decimal(90),
-  "Beetroot Seed": new Decimal(80),
-  "Cauliflower Seed": new Decimal(80),
-  "Parsnip Seed": new Decimal(60),
-  "Radish Seed": new Decimal(40),
-  "Wheat Seed": new Decimal(40),
-  "Kale Seed": new Decimal(30),
-
-  "Apple Seed": new Decimal(10),
-  "Orange Seed": new Decimal(10),
-  "Blueberry Seed": new Decimal(10),
-
-  Axe: new Decimal(125),
-  Pickaxe: new Decimal(50),
-  "Stone Pickaxe": new Decimal(10),
-  "Iron Pickaxe": new Decimal(5),
-  Shovel: new Decimal(1),
-  "Rusty Shovel": new Decimal(10),
-  "Power Shovel": new Decimal(5),
-  "Sand Shovel": new Decimal(25),
-  "Sand Drill": new Decimal(5),
-
-  Chicken: new Decimal(5),
-
-  "Magic Bean": new Decimal(5),
-  "Shiny Bean": new Decimal(5),
-  "Golden Bean": new Decimal(5),
-
-  "Immortal Pear": new Decimal(1),
+function isBuildingReady(building: PlacedItem[]) {
+  return building.some((b) => b.readyAt <= Date.now());
+}
+
+export const INITIAL_STOCK = (state?: GameState): Inventory => {
+  let tools = {
+    Axe: new Decimal(200),
+    Pickaxe: new Decimal(60),
+    "Stone Pickaxe": new Decimal(20),
+    "Iron Pickaxe": new Decimal(5),
+  };
+
+  // increase in 50% tool stock if you have a toolshed
+  if (
+    state?.buildings["Toolshed"] &&
+    isBuildingReady(state.buildings["Toolshed"])
+  ) {
+    tools = {
+      Axe: new Decimal(300),
```

Comment:
/nit
how about just using the initial stock value with multiplier, in that way you won't have to maintain 2 separate value and if in future initial stock changes it would be easy to implement
something like
`Axe: tools.Axe.times(1.5),`

Reply:
We are hardcoding these to have a clearer control over these amounts 👍